### PR TITLE
Port history.jelly from root history to modern ui

### PR DESCRIPTION
**Pull request by DuMaM**
Following previous changes I updated next jelly page.
https://github.com/jenkinsci/job-config-history-plugin/pull/199
https://github.com/jenkinsci/job-config-history-plugin/pull/202

image

image

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by DuMaM** [comment]
I don't know why but `show diff` button do not respects `flex-end` and adds small margin on right side.

**Review by NotMyFault — APPROVED** [review]
Looks great, thanks for your work!

**Comment by DuMaM** [comment]
@NotMyFault https://github.com/jenkinsci/job-config-history-plugin/pull/217#issuecomment-1183749364
Do you know how to remove this offset? It looks like button row is adding it.

**Comment by NotMyFault** [comment]
Where exactly do you mean?

**Comment by DuMaM** [comment]
> Where exactly do you mean?

https://user-images.githubusercontent.com/9051964/178848402-8dae4010-1399-4996-8564-9912320e4a8f.png
Take look on buttons  `show diff` and `restore project`, my aim was to align them to right. 
I was able to do this with `restore` one but somehow i wasn't able to do this with show diff.

**Comment by NotMyFault** [comment]
> 

Oic, I have nothing particular in mind, but I don't think that'd be a major blocker, considering this PR focuses on the table layout.

**Comment by DuMaM** [comment]
image

Now i'm happy with final results.
@NotMyFault You can merge it 😉

**Comment by DuMaM** [comment]
All this spaces where added by class - "jenkins-buttons-row"
I think there is bug similar to radio buttons, but i was unable to find it and fix it

**Comment by DuMaM** [comment]
@NotMyFault sorry i by mistake removed my fork 😶‍🌫️

File: src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/history.jelly [hunk 1]
```diff
@@ -66,174 +68,186 @@
...
+                      </td>
+
+                      <j:if test="${isDeleted}">
+                        <j:set var="offsetIfDeleted" value="1" />
+                      </j:if>
+
+                      <!-- File A -->
+                      <td style="text-align:center">
+                        <j:if test="${config.operation != 'Deleted'}">
+                          <f:radio name="timestamp1" value="${config.getDate()}" checked="${configNr == (2 + offsetIfFileMissing) ? true:false}" />
```
Comment (DuMaM) [inline src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistoryRootAction/history.jelly:123]:
```suggestion
                          <f:radio name="timestamp1" value="${config.getDate()}" checked="${configNr == (2 + offsetIfDeleted) ? true:false}" />

```

**Review by DuMaM — COMMENTED** [review]
I think i found one mistake

**Comment by DuMaM** [comment]
Ok I fixed it 
image

But somehow PR was not updated

**Comment by DuMaM** [comment]
Closed in favor of https://github.com/jenkinsci/job-config-history-plugin/pull/223
After removing fork Github was unable to add new commits to PR.

**Pull request closed without merging**